public/docs/ts/latest/guide/index.jade

```diff
@@ -120,10 +113,10 @@ block example-links
 
   # Feedback
 
-  We welcome feedback! Leave a comment by clicking the icon in upper right corner of the banner.
+  We welcome feedback! To leave a comment click the icon in the upper right corner of the banner.
```

Comment:
Oh, you're right. There used to be an icon for this, but isn't any longer. There's a "Report Issues" link in the footer, but that links to angular/angular (as reported in https://github.com/angular/angular.io/issues/2124).

Let's just delete this sentence, at least for now.